spring-restdocs-mockmvc/src/test/java/org/springframework/restdocs/mockmvc/MockMvcRestDocumentationIntegrationTests.java

```diff
@@ -221,8 +239,24 @@ public void httpieSnippetWithContent() throws Exception {
 				new File(
 						"build/generated-snippets/httpie-snippet-with-content/httpie-request.adoc"),
 				is(snippet(asciidoctor()).withContents(codeBlock(asciidoctor(), "bash")
-						.content("$ echo 'content' | http POST 'http://localhost:8080/'"
-								+ " 'Accept:application/json'"))));
+						.content(String.format("$ echo 'content' | http POST 'http://localhost:8080/'"
+								+ " \\%n 'Accept:application/json'")))));
+	}
+
+	@Test
+	public void httpieSnippetWithContentSpaceSeparator() throws Exception {
```

Comment:
It's good to have tests for the HTTPie snippet with different command formatters, but there's nothing MockMvc-specific about it so I don't think they belong here.